lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -89,15 +187,45 @@ def self.calc_next_release_base_version(version)
           { VERSION_NAME => "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}", VERSION_CODE => version[VERSION_CODE] }
         end
 
+        # Compute the name of the next version to use after code freeze, by incrementing the current version name and making it a `-rc-1`
+        #
+        # @example
+        #   calc_next_release_version({"name": "1.2", "code": 456}) #=> {"name":"1.3-rc-1", "code": 457}
+        #   calc_next_release_version({"name": "1.2.3", "code": 456}) #=> {"name":"1.3-rc-1", "code": 457}
+        #   calc_next_release_version({"name": "1.2", "code": 456}, {"name":"alpha-1.2", "code": 457}) #=> {"name":"1.3-rc-1", "code": 458}
+        #
+        # @param [Hash] version The current version hash, with keys `"name"` and `"code"`
+        # @param [Hash] alpha_version The current alpha version hash, with keys `"name"` and `"code"`, or nil if no alpha version
+        #
+        # @return [Hash] The hash containing the version name and code to use after release cut
+        #
         def self.calc_next_release_version(version, alpha_version = nil)
           nv = calc_next_release_base_version({ VERSION_NAME => version[VERSION_NAME], VERSION_CODE => alpha_version.nil? ? version[VERSION_CODE] : [version[VERSION_CODE], alpha_version[VERSION_CODE]].max})
           calc_next_beta_version(nv)
         end
 
+        # Compute the name and code of the next hotfix version.
+        #
+        # @param [String] hotfix_version_name The next version name we want for the hotfix
+        # @param [String] hotfix_version_code The next version code we want for the hotfix
+        #
+        # @return [Hash] The predicted next hotfix version, as a Hash containing the keys `"name"` and `"code"`
+        #
         def self.calc_next_hotfix_version(hotfix_version_name, hotfix_version_code)
-          { VERSION_NAME => hotfix_version_name, VERSION_CODE => hotfix_version_code}
+          { VERSION_NAME => hotfix_version_name, VERSION_CODE => hotfix_version_code }
```

Comment:
This one was also quite confusing, name suggesting we compute the next hotfix so that we will increment the versionName and code… but in practice we're not. Which matches how it's used at call site, but is damn confusingly and misleadingly named.

Will probably either rename the method to a more apt name in a subsequent PR, or more likely try to match `ios` here and update the call site to let this method do the hotfix increment as its name suggests, instead of making the increment at call site. That will require some refactoring of the Action tho, which is why I kept that for a future, separate PR.